firestore.rules

```diff
@@ -0,0 +1,13 @@
+rules_version = '2';
+service cloud.firestore {
+  match /databases/{database}/documents {
+    match /{document=**} {
+      allow read: if
+          request.time < timestamp.date(2021, 1, 1);
+    }
+    match /scholarships/{scholarshipId} {
+    allow write: if
+    	request.auth != null;
+    }
```

Comment:
Does prettier format these files? If not, then I would use the `{}` as a guide for where to indent:

```suggestion
    match /{document=**} {
      allow read: if request.time < timestamp.date(2021, 1, 1);
    }
    match /scholarships/{scholarshipId} {
      allow write: if request.auth != null;
    }
```